### PR TITLE
Add NETGEAR Switch Discovery Tool.app v1.1.115

### PR DESCRIPTION
**Pull request by ericpardee**
After making all changes to the cask:

- [x ] `brew cask audit --download {{cask_file}}` is error-free.
- [x ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x ] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x ] Named the cask according to the [token reference].
- [x ] `brew cask install {{cask_file}}` worked successfully.
- [x ] `brew cask uninstall {{cask_file}}` worked successfully.
- [x ] Checked there are no [open pull requests] for the same cask.
- [x ] Checked the cask was not already refused in [closed issues].
- [x ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-drivers/pulls
[closed issues]: https://github.com/caskroom/homebrew-drivers/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/netgear-switch-discovery-tool.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'netgear-switch-discovery-tool' do
+  version '1.1.115'
+  sha256 'e2988b8ebb58e8953885a46c4f9e94ba6ebebfc34b93cc902fbc44540d798bf8'
+
+  url "https://www.downloads.netgear.com/files/GDC/NetgearSDT-V#{version}.zip"
+  name 'netgear-switch-discovery-tool'
```
Comment (commitay) [inline Casks/netgear-switch-discovery-tool.rb]: `NETGEAR Switch Discovery Tool`

File: Casks/netgear-switch-discovery-tool.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  homepage 'https://www.netgear.com/support/download/'
```
Comment (commitay) [inline Casks/netgear-switch-discovery-tool.rb]: Can you change this to the page for this `app`?

**Comment by ericpardee** [comment]
He's the thing with that, I found this app when trying to update my own switch:
https://www.netgear.com/support/product/GS116Ev2.aspx#NETGEAR Switch Discovery Tool for Mac Version 1.1.115

However, on Netgear's site, that page for that app is set for each device, i.e.:
https://www.netgear.com/support/product/GS108Ev2.aspx#NETGEAR Switch Discovery Tool for Mac Version 1.1.115
https://www.netgear.com/support/product/GS108PEv3.aspx#NETGEAR Switch Discovery Tool for Mac Version 1.1.115
https://www.netgear.com/support/product/GS105E.aspx#NETGEAR Switch Discovery Tool for Mac Version 1.1.115
https://www.netgear.com/support/product/FS116E.aspx#NETGEAR Switch Discovery Tool for Mac Version 1.1.115
etc.

So, I wasn't able to find a non-device specific page for the app so I put 'https://www.netgear.com/support/download/'

What do you want me to do?

**Comment by commitay** [comment]
The `homepage` can stay as `https://www.netgear.com/support/download/`

**Comment by ericpardee** [comment]
anything else to get this merged?

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**